UnityProjects/MRTKDevTemplate/Packages/packages-lock.json

```diff
@@ -360,7 +360,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.xr.interaction.toolkit": {
-      "version": "2.0.2",
+      "version": "2.0.3",
```

Comment:
With this update, we may be able to remove our workaround for input sim from #10843

![image](https://user-images.githubusercontent.com/3580640/187260755-4740dd65-83a2-495d-9f94-6d59a3503a13.png)